Dockerfile

```
@@ -0,0 +1,19 @@
+# Stage 1: Build the application
+FROM node:20.19 AS build
+
+WORKDIR /app
+
+COPY package*.json ./
```

It's smart that you used a wildcard here to include both package.json and package-lock.json, but I also think that it could introduce other problems in the future, such as copying an unwanted file that matches the grep. As we're only saving a few chars here, I would suggest to just write out both filenames.

```suggestion
COPY package.json package-lock.json ./
```